Parser/StarmonMiniParse.m

```diff
@@ -314,7 +314,7 @@
     iContent = i + nColumnTime;
     
     switch header.param(iParam).column
-        case 'Temp(�C)'
+        case 'Temp0x280xB0C0x29' %degrees C
```

Comment:
Unfortunately Matlab textscan relies on fopen which default encoding scheme for reading files is platform dependent. It seems like it is UTF-8 for Linux and windows-1252 for Windows.